src/background/services/paymentSession.ts

```diff
@@ -53,7 +65,10 @@ export class PaymentSession {
     private requestId: string,
     private tabId: number,
     private frameId: number,
+    private storage: StorageService,
     private openPaymentsService: OpenPaymentsService,
+    private walletService: WalletService,
```

Comment:
Unused. (I wasn't sure why `WalletService` could be needed in `PaymentSession`, thankfully it's unused)